src/app/shop/[category]/[productId]/_components/Product/OptionWithButtons.tsx

```diff
@@ -12,6 +12,9 @@ import classNames from 'classnames/bind';
 import { useRouter } from 'next/navigation';
 import { useEffect, useState } from 'react';
 import { toast } from 'react-toastify';
+
+import { CartAPIDataType, ShopDataType } from '@/types/CartTypes';
```

Comment:
```suggestion
import type { CartAPIDataType, ShopDataType } from '@/types/CartTypes';
```

저도 이거 자주 빼먹어서,, 더 잘보이나봐요,,,😅